hack/build-go.sh

```diff
@@ -128,13 +131,16 @@ for arg in $args; do
 
             # build virtctl for all architectures if requested
             if [ "${BIN_NAME}" = "virtctl" -a "${KUBEVIRT_RELEASE}" = "true" ]; then
-                for arch in amd64 arm64; do
+                for arch in amd64 arm64 s390x; do
                     for os in linux darwin windows; do
                         if [ "${os}" = "windows" ]; then
                             extension=".exe"
                         else
                             extension=""
                         fi
+                        if [ "${os}" != "windows" ] && [ "${arch}" = "s390x" ]; then
+                            break
+                        fi
```

Comment:
That right there is the reason checking all the build scripts is on my todo list at the top.
I did a lot of changes in the scripts at the beginning, since there are a lot of options in the Makefile and i was unsure what i should be using.

But it is also wrong because looking at the code again, it should be `[ "${os}" != "linux" ]`, since as you pointed out, there is no windows (or darwin) on s390x. I'll change it.

Reply:
Oh, I get the intention now: we're building virtctl for all possible combinations of OS and architecture, and in the case of s390x only Linux makes sense. Gotcha.

You should use `continue` instead of `break` though, to keep things working even as the list of operating systems or architectures gets shuffled.

Reply:
I changed it to `continue`, will push the update together with the rebase